db4odemo/src/userinterface/UHCS/DoctorJPanel.java

```diff
@@ -41,19 +61,478 @@ public DoctorJPanel(JPanel userProcessContainer, UserAccount account, EcoSystem
+        complaintWorkRequest.setStatus("UHCS Doctor Accepted");
+        
+        refresh();
+
+        //enable buttons
+        /*btnStartTreatment.setEnabled(true);
+        btnAcceptCase.setEnabled(false);
+
+        //update tables
+        populateTable();*/
+    }//GEN-LAST:event_btnAcceptCaseActionPerformed
+
+    private void btnAssignAdvisorActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnAssignAdvisorActionPerformed
+        
+        if (txtDoctorFeedBack.getText().toString().isEmpty()) {
+            JOptionPane.showMessageDialog(null, "Feedback is mandatory!!");
+            return;
+        }
+        
+        btnCloseComplaint.setEnabled(false);
+        btnAssignAdvisor.setEnabled(false);
+        
+        Complaint complaintWorkRequest = (Complaint) tblComplaintsWithOpenStatus.getValueAt(0, 0);
+        complaintWorkRequest.setDoctorFeedback(txtDoctorFeedBack.getText().toString());
+        //status is set in next page
+        
+        //setting current complaint to null and freeing himself
+        doctor.setComplaint(null);
+        doctor.getEmployee().setIsFree(true);
+        account.getEmployee().setIsFree(true);
+        
+        // clear selected table
+        DefaultTableModel model = (DefaultTableModel) tblComplaintsWithOpenStatus.getModel();
+        model.setRowCount(0);
+        
+        disableAll();
+
+        //enable buttons
+        //btnStartTreatment.setEnabled(false);
+
+        //move to next page
+        AssignAdvisorJPanel page = new AssignAdvisorJPanel(userProcessContainer,account,complaintWorkRequest,system);
+        userProcessContainer.add("Assign Advisor Page",page);
+        CardLayout layout=(CardLayout)userProcessContainer.getLayout();
+        layout.next(userProcessContainer);
+
+        refresh();
+        //update tables - this will be done, when moved from next page
+        /*populateSelectedTable(complaintWorkRequest);
+        populateTable();*/
+    }//GEN-LAST:event_btnAssignAdvisorActionPerformed
+
+    private void btnStartTreatmentActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnStartTreatmentActionPerformed
+        btnStartTreatment.setEnabled(false);
+        
+        Complaint complaintWorkRequest = (Complaint) tblComplaintsWithOpenStatus.getValueAt(0, 0);
+        complaintWorkRequest.setStatus("UHCS Doctor Treating");
+        
+        refresh();
+    }//GEN-LAST:event_btnStartTreatmentActionPerformed
+
+    private void btnScheduleTreatmentActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnScheduleTreatmentActionPerformed
+        btnScheduleTreatment.setEnabled(false);
+        
+        Complaint complaintWorkRequest = (Complaint) tblComplaintsWithOpenStatus.getValueAt(0, 0);
+        complaintWorkRequest.setStatus("UHCS Doctor Scheduled Treatment");
+        
+        system.getComplaintDirectory().getComplaint(complaintWorkRequest.getComplaintID()).setNotifyFromDoctor(true);
+        
```

Comment:
victimStudent.
complaint= system.getComplaintDirectory().getComplaint(complaintWorkRequest.getComplaintID());
allStudentComplaints=complaint.getVictimStudent().getMyComplaints();

Reply:
resolved and pushed to this pr, in second commit of same pr